test/unit/org/apache/cassandra/index/sai/cql/InetAddressTypeEquivalencyTest.java

```diff
@@ -31,6 +33,15 @@
  */
 public class InetAddressTypeEquivalencyTest extends SAITester
 {
+    // TODO: Disables external execution because we know SAI indexing for inet works differently than RowFilter,
+    //  which can wrongly discard rows in the coordinator. This is reported in CNDB-12978, and we should enable
+    //  distributed execution again once we have a fix.
```

Comment:
Otherwise, we would have 51 failures on CI distracting us until we fix CNDB-12978: https://jenkins-stargazer.aws.dsinternal.org/job/ds-cassandra-pr-gate/job/PR-1564/7/testReport/